packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +26,22 @@ export function needsNxBadge(
 
   return false;
 }
+
+export function getActiveButtonIndex(
+  element: Element | null
+): [number, HTMLButtonElement[]] | undefined {
+  if (!element) {
+    return undefined;
+  }
+
+  const activeButton = document.activeElement;
+
+  if (!activeButton || !(activeButton instanceof HTMLButtonElement)) {
+    return undefined;
+  }
+
+  const buttonElementList = [...element.getElementsByTagName('button')]; // eslint-disable-line unicorn/prefer-query-selector
```

Comment:
If you're not taking advantage of the live `NodeList` (which is probably a good thing), then there's no point in using `getElementsByTagName` over `querySelectorAll`.